engine/render/src/render/render_script.cpp

```diff
@@ -1762,7 +1762,10 @@ namespace dmRender
      * @param [options] [type:table] optional table with properties:
      *
      * `frustum`
-     * : [type:vmath.matrix4] A frustum matrix used to cull renderable items. (E.g. `local frustum = proj * view`). May be nil.
+     * : [type:vmath.matrix4] A frustum matrix used to cull renderable items. (E.g. `local frustum = proj * view`). default=nil
+     *
+     * `frustum_skip_near_far`
+     * : [type:bool] Determines if frustum culling will use near and far planes of the frustum planes. default=true
```

Comment:
To me it feels a bit backwards to include using the near and far planes by setting "skip_near_far = false" instead of something like:

```
frustum_include_near_far = true
frustum_planes = render.FRUSTUM_ALL -- and some other mask for the other 'mode': render.FRUSTUM_XY
```

Reply:
Good point, I'll revise this.